src/components/common/Actions.vue

```diff
@@ -102,7 +102,8 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
 
   /** True if the File and Pay button should be disabled. */
   get isFilePayButtonDisabled (): boolean {
-    return (this.isBusySaving || !this.isCorrectionValid || this.isCorrectionEditing)
+    const isAllowEmptyCorrections = GetFeatureFlag('allow-empty-corrections')
+    return (this.isBusySaving || !this.isCorrectionValid || this.isCorrectionEditing || !isAllowEmptyCorrections)
```

Comment:
Don't you need to conditionally re-enable the checks that you commented out in the previous ticket?

Reply:
Outdated conversation. Resolving.